docs/contributing/technical/resolve-merge-conflicts.md

```diff
@@ -6,138 +6,79 @@ keywords:
 
-Running `git remote -v` will output:
-
-```shell
-origin git@github.com:0-vortex/open-sauced.git (fetch)
-origin git@github.com:0-vortex/open-sauced.git (push)
+```bash
+cd app
+git checkout <your-branch>
 ```
 
-As an additional step for this tutorial, we need to add the `upstream` remote:
+2. Fetch the latest changes from the OpenSauced main repository.
 
-```shell
-git remote add upstream git@github.com:open-sauced/open-sauced.git
+```bash
+git fetch upstream
 ```
 
-## Update
-
-First, get the default branch changes:
+3. See the differences between your branch and the OpenSauced main branch
 
-```shell
-git fetch origin --recurse-submodules=no --progress --prune
-git checkout main --
-git fetch upstream --recurse-submodules=no --progress --prune
-git merge upstream/main --no-stat -v
+```bash
+git diff <your-branch> upstream/main
 ```
 
-## Merge with `upstream`
-
-Then merge with the forked up-to-date `beta` (default branch):
-
-```shell
-git merge origin/main --no-ff -v
-```
-
-You will see something similar to:
-
-![proper merge but results in conflicts](../../../static/img/contributing-resolve-merge-conflicts-merge-conflicts.png)
+**NOTE**: Some of the OpenSauced repositories will use `main` for the main branch while others like the [app repository](https://github.com/open-sauced/app), will use `beta` for the main branch name.
 
-## Review Changes
+![git diff output review conflicts](../../../static/img/contributing-resolve-merge-conflicts-review-conflicts.png)
 
-To see what the changes are, run the command below:
+**NOTE**: If you have already opened up a pull request, then you can see if there is a conflict at the bottom of the PR. Even though it is possible to resolve conflicts through GitHub, it is best practice to resolve them locally.
 
-```shell
-git diff package.json
-```
-
-It will look like this:
+![merge conflicts git message](../../../static/img/contributing-resolve-merge-conflicts-dont-do.png)
 
-![review merge conflicts](../../../static/img/contributing-resolve-merge-conflicts-review-conflicts.png)
+## How to Resolve Merge Conflicts
 
-## Resolve Conflicts
+If you have merge conflicts, it is best to use a text editor to resolve them. Identify the conflicting files and open them up in your editor of choice. For the conflicting sections, you have an option to either keep the incoming changes, keep your changes or keep both sets of changes. The conflicting sections will be marked with `<<<<<<<`, `=======`, and `>>>>>>>` symbols.
 
-Since this pull request does not modify the `package.json` file, it is safe to fast-forward the changes from `origin/main`:
-
-```shell
-# overwrite with origin/main changes
-git show :3:package.json > package.json
-```
+Here is a [guide](https://dev.to/adiatiayu/how-to-resolve-merge-conflicts-using-the-merge-editor-feature-on-vs-code-pic) for an in depth walkthrough of the process. Once you have resolved all of the conflicts, then you can stage, commit and push your changes to your branch.
```

Comment:
Thanks for considering using my blog here! 🥰
But let's use @BekahHW's [post](https://dev.to/opensauced/keeping-your-branch-up-to-date-and-handling-merge-conflicts-while-waiting-for-pr-reviews-3b3h) as it's under OpenSauced's blog. 😊